YoCode/CommandLineArguments/CommandExtractor.cs

```diff
@@ -8,13 +9,16 @@ class CommandExtractor
     {
         public static SplitArg ArgsSplitter(string arg)
         {
+            var argParts = arg.Split(new[] { CommandIdentifiers.commandIdentifier, CommandIdentifiers.dataIdentifier }, StringSplitOptions.RemoveEmptyEntries);
             return new SplitArg
             {
-                command = (ExtractSuffix(arg,CommandIdentifiers.dataIdentifier) != null) 
-                ? ExtractPrefix(arg.Substring(0, arg.IndexOf(CommandIdentifiers.dataIdentifier)), CommandIdentifiers.commandIdentifier) 
-                : ExtractPrefix(arg, CommandIdentifiers.commandIdentifier),
+                //command = (ExtractSuffix(arg,CommandIdentifiers.dataIdentifier) != null) 
+                //? ExtractPrefix(arg.Substring(0, arg.IndexOf(CommandIdentifiers.dataIdentifier)), CommandIdentifiers.commandIdentifier) 
+                //: ExtractPrefix(arg, CommandIdentifiers.commandIdentifier),
```

Comment:
Why are you keeping the commented out code? Git has it in its history if you ever need it back.

Reply:
Old habits 🙁